### ISSUE TITLE
Migrate XAML language services to the external access layer pattern

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#35069]

**Comment by mgoertz-msft** [comment]
@sharwell, Microsoft.VisualStudio.LanguageServices.Xaml is an external access assembly - in fact it was the very first one. It does not give access to the internals of Roslyn but rather only access to the internals of this access assembly. It was created exactly for this purpose so that we don't need IVT access to Roslyn.

**Comment by sharwell** [comment]
@mgoertz-msft Yes, once the assembly is updated to match the External Access assembly naming conventions it will stop getting flagged by Build Boss.
